osquery/database/database.cpp

```diff
@@ -527,11 +526,11 @@ static Status migrateV1V2(void) {
   }
 
   for (const auto& key : keys) {
-    std::smatch match;
-    if (std::regex_match(key, match, re)) {
+    const auto pos = key.find(audit_str);
```

Comment:
nit: maybe don't define `audit_str` and just put the constant here like we use `".auditeventpublisher."`.

Reply:
.audit. is used twice(once string itself and once length in the replace), so I think it's more intuitive